src/components/CoinDetails.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { useLocation, Link } from 'react-router-dom';
+import { BsX } from 'react-icons/bs';
+
+const CoinDetails = () => {
+  const {
+    state: { crypto },
+  } = useLocation();
+  return (
+    <div className="container">
```

Comment:
- [ ] Please fix the horizontal scrolling on the details page.

https://user-images.githubusercontent.com/24703171/182432173-8338d2c4-3043-4a12-8916-c81b3ef74ce2.mp4